extensions/guacamole-auth-saml/src/licenses/LICENSE

```diff
@@ -0,0 +1,257 @@
+      whether in tort (including negligence), contract, or otherwise,
+      unless required by applicable law (such as deliberate and grossly
+      negligent acts) or agreed to in writing, shall any Contributor be
+      liable to You for damages, including any direct, indirect, special,
+      incidental, or consequential damages of any character arising as a
+      result of this License or out of the use or inability to use the
+      Work (including but not limited to damages for loss of goodwill,
+      work stoppage, computer failure or malfunction, or any and all
+      other commercial damages or losses), even if such Contributor
+      has been advised of the possibility of such damages.
+
+   9. Accepting Warranty or Additional Liability. While redistributing
+      the Work or Derivative Works thereof, You may choose to offer,
+      and charge a fee for, acceptance of support, warranty, indemnity,
+      or other liability obligations and/or rights consistent with this
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   APPENDIX: How to apply the Apache License to your work.
+
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+
+==============================================================================
+
+APACHE GUACAMOLE SUBCOMPONENTS
+
+Apache Guacamole includes a number of subcomponents with separate copyright
+notices and license terms. Your use of these subcomponents is subject to the
+terms and conditions of the following licenses.
```

Comment:
I think some licenses are missing here and within `src/licenses/bundled/`. Checking with Maven:

```
[mjumper@dev-mjumper guacamole-auth-saml]$ mvn dependency:tree -Dscope=runtime
[INFO] Scanning for projects...
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Building guacamole-auth-saml 1.2.0
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-dependency-plugin:2.10:tree (default-cli) @ guacamole-auth-saml ---
[INFO] org.apache.guacamole:guacamole-auth-saml:jar:1.2.0
[INFO] +- com.google.inject:guice:jar:3.0:compile
[INFO] |  +- javax.inject:javax.inject:jar:1:compile
[INFO] |  \- aopalliance:aopalliance:jar:1.0:compile
[INFO] +- com.google.inject.extensions:guice-multibindings:jar:3.0:compile
[INFO] \- com.onelogin:java-saml:jar:2.5.0:compile
[INFO]    +- com.onelogin:java-saml-core:jar:2.5.0:compile
[INFO]    +- joda-time:joda-time:jar:2.10.3:compile
[INFO]    +- org.apache.commons:commons-lang3:jar:3.4:compile
[INFO]    +- org.apache.santuario:xmlsec:jar:2.1.4:compile
[INFO]    |  \- com.fasterxml.woodstox:woodstox-core:jar:5.0.3:compile
[INFO]    |     \- org.codehaus.woodstox:stax2-api:jar:3.1.4:compile
[INFO]    \- commons-codec:commons-codec:jar:1.12:compile
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 3.839 s
[INFO] Finished at: 2020-06-23T14:00:00-07:00
[INFO] Final Memory: 14M/174M
[INFO] ------------------------------------------------------------------------
[mjumper@dev-mjumper guacamole-auth-saml]$ 
```

So, looks like Apache Commons Codec, Apache Commons Lang, Apache Santuario, Joda-Time, and Woodstox?

Reply:
Okay, I think I got everything added properly - both to overall license file and the bundled directory.